api/structure/schema.py

```diff
@@ -0,0 +1,45 @@
+import graphene
+from graphene_sqlalchemy import SQLAlchemyObjectType
+
+from api.structure.models import Structure as StructureModel
+from helpers.auth.authentication import Auth
+from helpers.structure.create_structure import create_structure
+
+
+class Structure(SQLAlchemyObjectType):
+    """Autogenerated Return type for OfficeStructure"""
+
+    class Meta:
+        model = StructureModel
+
+
+class StructureInputs(graphene.InputObjectType):
+    web_id = graphene.String(required=True)
```

Comment:
Change this to `structure_id` as agreed upon during the previous sync.

Reply:
Add documentation for the mutation you have added

Reply:
Hello @Bonifase , a task was created to rename the `webId` to `structureId` on the office structure model. I will need that task to be completed, approved and merged before I can complete this one.